com.unity.render-pipelines.high-definition-config/Runtime/ShaderConfig.cs

```diff
@@ -20,6 +20,22 @@ public enum ProbeVolumesEvaluationModes
         MaterialPass = 2,
     }
 
+    [GenerateHLSL(PackingRules.Exact)]
+    public enum ProbeVolumesEncodingModes
+    {
+        SphericalHarmonicsL0 = 0,
+        SphericalHarmonicsL1 = 1,
+        SphericalHarmonicsL2 = 2
+    }
+
+    [GenerateHLSL(PackingRules.Exact)]
+    public enum ProbeVolumesBilateralFilteringModes
```

Comment:
Just something to have in mind: the more you have option in shader config, the less they are tested (as it required a dedicated project to test it).

Reply:
Yeah that's a good point. I've been leaning in to using ShaderConfig here to avoid creating more shader variants, as I know the variant explosion has been a big problem. But yeah, good point to at least be aware that that means less test coverage.